sdk/communication/azure-communication-networktraversal/src/main/java/com/azure/communication/networktraversal/models/CommunicationIceServerRouteType.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.networktraversal.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for CommunicationIceServerRouteType. */
+public final class CommunicationIceServerRouteType extends ExpandableStringEnum<CommunicationIceServerRouteType> {
```

Comment:
Why do we have 2 enum types with the same values? Do we expect the values of these types to diverge in the future?

Reply:
In the swagger we only define it once, but after running the command for autogenerated code it creates two different. Is there a way to say, "Only create one"?